### ISSUE TITLE
Physics Range Extender doesn’t work.

### ISSUE THREAD

**Issue by Red1949** [BrettRyland/BDArmory#454]
Yesterday I’m noticed when that vessels that are more far than 200m disappears these vehicles’ icon and it disappears in BDA switch.
I have all required files (Module Manager and Physics Range extender), I’m even removed all mods except BDA, PRE and MM, it doesn’t help.
Playing a steam license KSP, lastest version.
P.S: Sorry for my Eng.

**Comment by BrettRyland** [comment]
First, make sure that PRE is enabled and set to reasonable values, e.g., the defaults give a 100km range
![Screenshot_20221120_120618](https://user-images.githubusercontent.com/18440126/202898744-b61ac244-1d19-422e-88b5-90c7febd6c49.png)
and that the KSP.log file doesn't have any exceptions (entries with `EXC` by the timestamp) related to PRE.

Even if PRE was disabled, however, vessels still shouldn't disappear beyond 200m. The default unload distance in KSP is around 20km, I think. So, there's likely some other issue causing this, such as bystander removal during competitions or aircraft that are landed too long during competitions. For these you will need to enable debugging in the BDA settings
![Screenshot_20221120_122144](https://user-images.githubusercontent.com/18440126/202899352-791c088a-460e-42be-a7ad-2e80ab74125b.png)
and then examine the KSP.log for entries such as
```
[LOG 12:21:27.590] [BDArmory.BDACompetitionMode]: ...
```

**Comment by Red1949** [comment]
Okay, thanks, I'll try it.

**Comment by Red1949** [comment]
Well, I have some free time so…
Few days ago I’m reinstalled KSP by deleting ALL files except Steam Cloud files like saves and etc.
Maybe I’m explained everything to you wrong. Disappeared not vessel itself, I mean HUD (Icon of vehicle) of this vehicle is disappearing, EVEN VANILLA ONE. Well, when I reinstalled KSP, I fixed vanilla HUD, now only BDA HUD disappearing.
I noticed this problem appeared when 1.12.4 was released and when I turned off Squad launched by steam launch option. Today I will try to play on 1.12.3. and play with launcher to check what one of these causes problem and tell you about it. Thanks a lot.
Sorry for my eng again.
